guides/source/asset_pipeline.md

```diff
@@ -33,13 +33,10 @@ passing the `--skip-sprockets` option.
 rails new appname --skip-sprockets
 ```
 
-Rails automatically adds the `sass-rails`, `coffee-rails` and `uglifier`
-gems to your `Gemfile`, which are used by Sprockets for asset compression:
+Rails automatically add the `sass-rails` gem to your `Gemfile`, which are used by Sprockets for asset compression:
 
 ```ruby
 gem 'sass-rails'
-gem 'uglifier'
-gem 'coffee-rails'
 ```
 
 Using the `--skip-sprockets` option will prevent Rails from adding
```

Comment:
👋

I was reading this guide and noticed a bit of confusing wording on the two lines following this one (unfortunately GitHub won't let me comment directly on those lines). After looking through the history I found that the wording became confusing when this PR removed the mentions of `uglifier` and `coffee-rails`, leaving only `sass-rails`.

So, I've gone ahead and opened https://github.com/rails/rails/pull/38782 as a follow-up to address this.